packages/perseus/src/__tests__/widgets.test.ts

```diff
@@ -12,19 +12,8 @@ describe("Widget API support", () => {
     // validate function. Not all widgets support this function so even though
     // this list looks exhaustive, it's not!
     it.each([
-        "deprecated-standin",
```

Comment:
Over the last little while I've been thinking that perhaps our `WidgetExports<T>` should be where a widget can describe if it's scorable or not.  In doing so, the type would require all the necessary functions needed to do scoring (such as `getUserInput()`).  

No action needed and I think your idea of attaching the validators to each `WidgetExports<T>` gets us partially there. Just thinkin' aloud.